syntax= "proto3";

package ocean;

option cc_generic_services = true; // 配置后才会处理生成service

message ResultCode{
    int32 errcode=1;
    bytes errmsg=2;
}

message LoginRequest{
    bytes name=1;
    bytes pwd=2;
}

message LoginResponse{
    ResultCode res=1;
    bool success=2;
}

service UserServiceRpc{
    rpc Login(LoginRequest) returns(LoginResponse);
}